Dockerfile

```
@@ -0,0 +1,8 @@
+FROM ruby:alpine
+WORKDIR /code
+COPY . /code
+RUN apk update && apk add --virtual build-dependencies build-base ruby-dev libpq postgresql-dev tzdata
+RUN bundle config set --local without 'development test'
+RUN bundle install 
+CMD rails s -b 0.0.0.0 -p 80 -e production
+
```

Dependencies are way less likely to change than the source code, and so it may be a good idea to put them close to the top so that docker can cache layers efficiently
```suggestion
WORKDIR /code
RUN apk update && apk add --virtual build-dependencies build-base ruby-dev libpq postgresql-dev tzdata
RUN bundle config set --local without 'development test'
COPY Gemfile Gemfile.lock ./
RUN bundle install 
COPY . /code
CMD rails s -b 0.0.0.0 -p 80 -e production
```